### DISCUSSION TITLE
TrenchBroom Profile Issue

### DISCUSSION THREAD

**Discussion by Mx-Crowe** [nzp-team/nzportable#676] [General]
I'm trying to get Trenchbroom working with the NZ:P Profile that's on the landing site. When I tried to restart Trenchbroom to get it working and load the compiler, it gave me this error message: 

"Some game configurations could not be loaded. The following errors occurred:
Failed to load game configuration file 'nzp\GameConfig.cfg': At line 2, column 16: Unsupported game configuration version 5; valid versions are: 7, 8"

I even re-downloaded both Trenchbroom and the NZ:P Trenchbroom profile to try and get it working, but nothing I've tried has worked. Any help would be appreciated. I've also attached an image of the error message in case that helps.
![Capture](https://github.com/nzp-team/nzportable/assets/162517385/44295944-71dc-4bb2-8827-f30f77d97fae)

**Comment by MotoLegacy** [comment]
Thanks, addressed with https://github.com/nzp-team/trenchbroom-profile/commit/d7bafd5d6ebedc2803971c0376d9a33c0ce63dac

**Reply by Mx-Crowe** [reply]
I tried changing it to read Version 7 already. Sorry, I should have put that in my original post. When I did that (and with the new update) I get this error message:

![Capture](https://github.com/nzp-team/nzportable/assets/162517385/874240a7-36aa-42a2-8f4a-cf9c49c84d97)

**Reply by Mx-Crowe** [reply]
Should I create an Issue under Bug for this since this isn't really a discussion post as much as a bug report? Sorta realized I should have done that instead of the Discussion post in retrospect.
